schemas/com.snowplowanalytics.snowplow.badrows/snowflake/jsonschema/1-0-0

```diff
@@ -0,0 +1,566 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Snowflake Loader specific bad row",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.badrows",
+    "name": "snowflake",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "anyOf": [
```

Comment:
I think this should be `oneOf`. `anyOf` means that both schemas could be applied, e.g. object could be with `original`, `errors` *and* `failures`, `enrichedEvent` properties, whereas we clearly want it to be only one of these sets.